core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -295,6 +304,16 @@ object AclCommand {
       .describedAs("broker")
       .ofType(classOf[Int])
 
+    val transactionalIdOpt = parser.accepts("transactional-id", "The transactionalId to which ACLs should " +
+      "be added or removed. A value of * indicates the ACLs should apply to all transactionalIds.")
+      .withRequiredArg
+      .describedAs("transactional-id")
+      .ofType(classOf[String])
+
+    val idempotentOpt = parser.accepts("idempotent", "Enable idempotence for the producer. This should be " +
+      "used in combination with the --producer option. Note that idempotence is enabled automatically if " +
```

Comment:
It seems this option can be used by itself and doesn't have to be used together with --producer?